clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -159,7 +226,8 @@ record_filter_t::process_memref(const trace_entry_t &memref)
 bool
 record_filter_t::print_results()
 {
-    // TODO i#5675: Print stats about filtered entries.
+    std::cerr << "Outputted " << output_entry_count_ << " entries from "
```

Comment:
nit: I think the past tense of "output" is just "output".  Apparently a search shows that "outputted" is also considered an alternative: it sounds wrong to me though; maybe regional dialects.

Reply:
Changed it. I was on the fence and made it as such just because of a quick search.